.github/ISSUE_TEMPLATE/issue report.yml

```diff
@@ -87,46 +144,42 @@ body:
     validations:
       required: true
   - type: markdown
+    id: general-information-logs
     attributes:
       value: |
-        When providing logs, please keep the following things in mind.
+        When providing logs, please keep the following things in mind:
         1. **DO NOT** use external paste services.
         2. Please provide complete logs.
           - For server logs, include everything you think is important plus *10 lines before and after*
           - For ffmpeg logs, please provide the entire file unmodified.
         3. Please do not run logs through any translation program. Especially beware if your browser translates pages by default.
```

Comment:
```suggestion
        3. Please do not run logs through any translation program. The original logs should be in English. If they appear as other languages in your browser, your browser is translating it. Please check your browser configuration and/or download the raw file from the Jellyfin server and view it in a text editor.
```

Reply:
I think that is too much text. "Please provide the raw untranslated logs."